lib/blacklight/models/content.rb

```diff
@@ -9,7 +9,9 @@ def initialize(xml)
     end
 
     def canvas_conversion
-      "<a href='$IMS_CC_FILEBASE$/#{@linkname}'>#{@linkname}</a>"
+      # rubocop:disable LineLength
```

Comment:
`oneLine.amountStupid() < addExrtaFunctionToMakeOneLine.amountStupid()` I ran that in the rails console, It evaluates to `true`